app/_src/gateway/plugin-development/pdk/kong.client.tls.md

```diff
@@ -32,30 +32,65 @@ Requests the client to present its client-side certificate to initiate mutual
  To find out whether the client honored the request, use
  `get_full_client_certificate_chain` in later phases.
 
+{% if_version gte:3.2.x %}
+The `ca_certs` argument is the optional CA certificate chain opaque pointer,
+ which can be created by the [parse_pem_cert](https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/ssl.md#parse_pem_cert)
+ or [resty.opensslx509.chain](https://github.com/fffonion/lua-resty-openssl#restyopensslx509chain)
+ The Distinguished Name (DN) list hints of the CA certificates will be sent to clients.
+ If omitted, will not send any DN list to clients.
 
 **Phases**
+{% endif_version %}
 
 * certificate
 
+{% if_version gte:3.2.x %}
+**Parameters**
+
+* **ca_certs** (`cdata`, _optional_):  The CA certificate chain opaque pointer
+{% endif_version %}
+
 **Returns**
 
+{% if_version lte:3.1.x %}
 1.  `true|nil`:  Returns `true` if request is received, or `nil` if
  request fails.
-
 1.  `nil|err`:   Returns `nil` if the handshake is successful, or an error
  message if it fails.
+{% endif_version %}
 
-
+{% if_version gte:3.2.x %}
+1.  `true|nil`:  Returns `true` if successful, or `nil` if it fails.
+1.  `nil|err`:  Returns `nil` if successful, or an error message if it fails.
+{% endif_version %}
```

Comment:
Can't leave a GH suggestion, but this is just a rephrase for clarity. You don't need to version tag it; just use the new text.